src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonReviewAside/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { Stack } from "@mui/material";
+
+import Polygons, { IpolygonFromMap, IPolygonItem } from "../Polygons";
+
+interface SitePolygonReviewAsideProps {
+  data: IPolygonItem[];
+  polygonFromMap?: IpolygonFromMap;
+  setPolygonFromMap?: any;
+  refresh?: () => void;
+  mapFunctions: any;
+}
+
+const SitePolygonReviewAside = (props: SitePolygonReviewAsideProps) => {
+  return (
+    <Stack gap={8} className="h-full">
```

Comment:
Please avoid the extra return statement for single-statement arrow functions to reduce indentation:
```
const SitePolygonReviewAside = (props: SitePolygonReviewAsideProps) => (
  <Stack gap={8} className="h-full">
    ...
  </Stack>
);
```